lsp-completion.el

```diff
@@ -675,8 +676,9 @@ The CLEANUP-FN will be called to cleanup."
   :lighter ""
   (let ((completion-started-fn (lambda (&rest _)
                                  (setq-local lsp-inhibit-lsp-hooks t)))
-        (after-completion-fn (lambda (&rest _)
-                               (lsp-completion--clear-cache)
+        (after-completion-fn (lambda (result)
+                               (when (stringp result)
```

Comment:
This would mean that, when aborting, the cache wouldn't be cleared. That might be better though, because if the user auto-completes again at the same point, the result will be cached.

Reply:
Yes, beside we have a cache clear again at completion start so the cache should not be leaked to become too large anyway.